config/directconnect/config.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2022 Upbound Inc.
+*/
+
+package directconnect
+
+import (
+	"github.com/upbound/upjet/pkg/config"
+)
+
+// Configure adds configurations for directconnect group.
+func Configure(p *config.Provider) { // nolint:gocyclo
+	p.AddResourceConfigurator("aws_dx_public_virtual_interface", func(r *config.Resource) {
+		r.References["connection_id"] = config.Reference{
+			Type: "github.com/upbound/provider-aws/apis/directconnect/v1beta1.Connection",
+		}
+		r.UseAsync = true
```

Comment:
The default mode is [async](https://github.com/upbound/upjet/blob/2bd418a5521b5a4a575c0fae7185bd7f60a06560/pkg/config/common.go#L71) in Upjet:
```suggestion
```

Reply:
Done